content/projects/kotlin/project-10/_index.md

```diff
@@ -1,5 +1,4 @@
 ---
-_db_id: 490
```

Comment:
If the db id is already there, please dont remove it.

Reply:
It's not meant to be there in this case, That was a result of me copying the template.

Reply:
Nope. 490 is in the master branch already. This wouldn't show up in the diff if there was no difference.

https://github.com/Umuzi-org/tech-department/blob/master/content/projects/kotlin/project-10/_index.md